examples/misc/coords_report.py

```diff
@@ -3,22 +3,23 @@
 Coords Report
 =============
 
-Override the default reporting of coords.
+Override the default reporting of coords in the interactive backend
+(this will not change the formatting of the yaxis tick labels).
```

Comment:
Maybe: 

```suggestion
Override the default reporting of coords as the mouse moves over the axes 
in an interactive backend.
```

I also don't understand your parenthetical comment, and suspect it would be more mystifying than helpful.

Reply:
Thanks, this sounds much better.  

The parenthetical comment was just meant to point out that `fmt_ydata` is used to format the coordinates in an interactive backend, not in the plot itself, as one might think at first glance when you come across this function for the first time (as it obviously was the case in #2205).